test/index.spec.ts

```diff
@@ -1070,7 +1070,7 @@ describe('[ TOTP ]', () => {
               reason.headers.get('set-cookie') ?? '',
             )
             expect(session.get(AUTH_OPTIONS.sessionErrorKey)).toEqual({
-              message: ERRORS.EXPIRED_TOTP,
+              message: ERRORS.TOTP_NO_EMAIL,
```

Comment:
Not sure if we want to replace `EXPIRED_TOTP`, but instead add a specific case for email not in session.

Reply:
I added another test for when the TOTP is not in the session, and made it explicit that the email and TOTP are the ones being unset (v.s. just an empty session). Does that work?